tests/unit/CMakeLists.txt

```
@@ -309,6 +309,10 @@ seastar_add_test (ipv6
 seastar_add_test (json_formatter
   SOURCES json_formatter_test.cc)
 
+seastar_add_test (kafka_protocol_test
```

NIT: I think the naming convention for adding test files "name_test.cc" is to leave only "name" in cmake name, i.e. `seastar_add_test (name ...)`, therefore:
```suggestion
seastar_add_test (kafka_protocol
```

---

Fixed in: https://github.com/psarna/seastar/pull/17/commits/55034090d38cc1c5567fddea825364d2859add57#diff-90e79c2c9c9258ee3a1bbac766e6c402R312